python-dateutil/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.5" %}
```

Comment:
This is a really old version used only by `WOFpy`, so I did not sent this one over to `conda-forge`.